src/screens/Home/index.tsx

```diff
@@ -32,6 +35,7 @@ const HomeScreen: React.FC<HomeScreenProps> = ({ appName }) => {
           alert("my nem jeff");
```

Comment:
don't wanna say but should we remove?

Reply:
haha they're just placeholders for showing our resusable components

Reply:
i assume tomas will remove when he starts on the home component